packages/eslint-plugin-nx/src/configs/angular.ts

```diff
@@ -16,6 +16,13 @@ export default {
     node: true,
   },
   plugins: ['@angular-eslint'],
-  extends: ['plugin:@angular-eslint/recommended'],
+  extends: [
+    'plugin:@angular-eslint/recommended',
+    /**
+     * TODO: Consider dropping this extends and explicitly carrying over rules we care about
+     * into our typescript preset in v13
+     */
+    'plugin:@angular-eslint/recommended--extra',
```

Comment:
Users might not have the latest version if they don't run the migration.

Can we add a check to only include this it exists in the version the user has installed?